### DISCUSSION TITLE
Using leaflet plugins

### DISCUSSION THREAD

**Discussion by MikeLacroix** [vue-leaflet/vue-leaflet#209] [Q&A]
Hello, i would like to use this leaflet plugin: [https://github.com/ppete2/Leaflet.PolylineMeasure](https://github.com/ppete2/Leaflet.PolylineMeasure). However I have no idea how to load it into vue-leaflet. Any hints? Thanks!

**Comment by gpproton** [comment]
I'm also looking for a solution for this

**Comment by devick** [comment]
I'm also looking for a solution for this

**Comment by xnhp** [comment]
See also [this thread](https://github.com/vue-leaflet/vue-leaflet/discussions/205) -- seems that a nice way to integrate leaflet plugins is still on the roadmap.
